docs/tools/vdb_table/data/aperturedb.json

```diff
@@ -67,93 +67,93 @@
+    "source_url": "https://docs.aperturedata.io/HowToGuides/Advanced/Image%20Similarity%20Search",
+    "comment": "Supports multimodal embeddings in general including for videos."
   },
   "embeddings_structured": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "partial",
+    "source_url": "https://docs.aperturedata.io/query_language/Reference/descriptor_commands/desc_commands/AddDescriptor",
+    "comment": "ApertureDB vector store can store embeddings from any source but we don't generate them ourselves."
   },
   "rag": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "full",
+    "source_url": "https://docs.aperturedata.io/category/chatbot-with-langchain-and-aperturedb",
+    "comment": "You can build hybrid-RAG with our graph-vector search."
   },
   "recsys": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "partial",
+    "source_url": "https://docs.aperturedata.io/HowToGuides/Applications/similarity_search",
+    "comment": "Recommendations can be built using our KNN and metadata search support."
   },
   "langchain": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://docs.aperturedata.io/Integrations/langchain_howto",
     "comment": ""
   },
   "llamaindex": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "full",
+    "source_url": "https://github.com/aperture-data/llama_index",
+    "comment": "Yet to contribute upstream but deployed with customer"
   },
   "managed_cloud": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "full",
+    "source_url": "https://cloud.aperturedata.io/",
+    "comment": "VPC options and community edition available as well"
   },
   "pricing": {
-    "value": "",
-    "source_url": "",
-    "comment": ""
+    "value": "full",
+    "source_url": "http://aperturedata.io/pricing",
+    "comment": "Offers free trial"
   },
   "in_process": {
     "support": "none",
     "source_url": "",
     "comment": ""
   },
   "multi_tenancy": {
-    "support": "",
+    "support": "full",
     "source_url": "",
-    "comment": ""
+    "comment": "Not documented externally."
   },
   "disk_index": {
     "support": "",
     "source_url": "",
     "comment": ""
   },
   "ephemeral": {
-    "support": "",
+    "support": "full",
     "source_url": "",
-    "comment": ""
+    "comment": "Not documented externally."
   },
   "sharding": {
-    "support": "",
+    "support": "full",
     "source_url": "",
-    "comment": ""
+    "comment": "Not documented externally."
   },
   "doc_size": {
     "bytes": 0,
-    "unlimited": false,
-    "source_url": "",
-    "comment": ""
+    "unlimited": true,
+    "source_url": "https://docs.aperturedata.io/administration/Distributed%20aperturedb",
+    "comment": "Multimodal data stored in a horizontally scalable ApertureDB deployment, not limited in count of objects."
```

Comment:
This field is not about count of objects, it is about the object size. Can I store a 10GB video file in aperturedb? If yes, then  keep it unlimited, but let's add any relevant caveats about performance into comment.